docs/config/index.md

```diff
@@ -88,7 +88,7 @@ Vite will process inlined modules. This could be helpful to handle packages that
 - **Type** `boolean`
 - **Default:** `false`
 
-When a dependency is a valid ESM package, try to guess the cjs version based on the path.
+When a dependency is a valid ESM package, try to guess the cjs version based on the path. This might be helpful, if dependency has the wrong ESM file.
```

Comment:
```suggestion
When a dependency is a valid ESM package, try to guess the cjs version based on the path. This might be helpful, if a dependency has the wrong ESM file.
```